### ISSUE TITLE
Registration Security Update

### ISSUE THREAD

**Issue by marchershey** [marchershey/lodge-board-beta#9]
As of right now, if a user types in an email address that already exists in the database, the error message "an account with this email address already exists" which in my opinion is a bad security practice. 

To combat this, when a user tries to register an account with an email address that is already assigned to an existing account, make it seem like the registration went through successfully and display the "email verification" page. But instead of sending the verification email, send a "Forgot password" email, or maybe a "Someone just tried to use your email."

I believe this will prevent malicious users from verifying whether or not emails exist in the database.

**Comment by marchershey** [comment]
Putting on hold - waiting on release check
